lib/pbench/server/api/resources/query_apis/iteration_samples.py

```diff
@@ -171,10 +185,10 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
 
     def postprocess(self, es_json: JSON, context: CONTEXT) -> JSON:
         """
-        Returns a non-empty list of aggregated unique values for keyword fields
-        in the 'benchmark', 'sample', 'iterations' and 'run' subdocuments of
-        result-data-sample index. The output only contains fields that has at
-        least 1 unique value.
+        Returns a stringified JSON object containing keyword/value pairs where
```

Comment:
Actually, this function returns a Flask `Response`, I think (see [jsonify()](https://flask.palletsprojects.com/en/2.0.x/api/#flask.json.jsonify)), so both the comment and the type hint could use some adjustment.

Reply:
Yeah and that's true for most of our APIs where the postprocess type-hint returns are JSON which are actually JSONified flask response object